src/Wrapper.js

```diff
@@ -11,9 +12,35 @@ import { getAppPath } from './routing'
 import { staticApps } from './static-apps'
 import { addressesEqual } from './web3-utils'
 import { noop } from './utils'
-import { APPS_STATUS_LOADING } from './symbols'
+import {
+  APPS_STATUS_ERROR,
+  APPS_STATUS_READY,
+  APPS_STATUS_LOADING,
+} from './symbols'
 
 class Wrapper extends React.Component {
+  static propTypes = {
+    account: PropTypes.string.isRequired,
+    apps: PropTypes.array.isRequired,
+    appsStatus: PropTypes.oneOf([
+      APPS_STATUS_ERROR,
+      APPS_STATUS_READY,
+      APPS_STATUS_LOADING,
+    ]).isRequired,
+    banner: PropTypes.oneOfType([PropTypes.bool, PropTypes.element]).isRequired,
+    connected: PropTypes.bool.isRequired,
+    daoAddress: PropTypes.string.isRequired,
+    historyBack: PropTypes.func.isRequired,
+    historyPush: PropTypes.func.isRequired,
+    locator: PropTypes.object.isRequired,
+    onRequestAppsReload: PropTypes.func.isRequired,
+    permissionsLoading: PropTypes.bool.isRequired,
+    transactionBag: PropTypes.object,
```

Comment:
It might be nice moving the non-required prop type declarations into their own group here, since there's so many props

Reply:
What do you mean their own group? Something like:

```js
const REQUIRED_PROPS = { /**/ }

static propTypes = {
  ...REQUIRED_PROPS,
  /* non required props */
 /* ... */
}
```

Reply:
Oh, just ordering them so that the required ones go first :).

Reply:
These are all sorted alphabetically, I'd prefer to keep it that way unless you have strong preferences for the order you propose.

Reply:
Let's leave it like this :).